requirements.txt

```diff
@@ -3,7 +3,6 @@ Flask-HTTPAuth==2.5.0
 Tempita==0.5dev
 boto==2.42.0
 coverage==4.2.0
-flake8==3.0.4
 gunicorn==19.9.0
 mock==1.0.1
```

Comment:
Really, this shouldn't be part of `install_requires` either...

Reply:
I'll try to split requirements in another PR

Reply:
I plan to split deps to base_requirements.txt and test_requirements.txt. set base_requirement in setup.require and test_requirements in setup.test_require. and requirements.txt include both files

Reply:
> I plan to split deps to base_requirements.txt and test_requirements.txt.

Why do we need that?  I would follow our usual procedure.

Reply:
set all requirements(exclude test requirements) in setup.pu?

Reply:
Yup. That's what we usually do.  It allows us to use specific versions for testing and a range to be installed along with the lib.

Reply:
What is the process to build virtualenv. As I see firstly we install yodeploy and then in separate command we install <service>/requirements.txt. Is it right? In this case setting range for requirements doesn't do effect.

Reply:
It's probably the same as with any other lib, nope?